_data/banking.yml

```diff
@@ -288,9 +288,12 @@ websites:
     - name: Commonwealth Bank of Australia
       url: https://www.commbank.com.au/
       img: commbank.png
-      tfa: No
-      twitter: CommBank
-      facebook: commonwealthbank
+      tfa: Yes
+      sms: Yes
+      software: Yes
```

Comment:
I don't see any mentions of a software based 2FA solution for the documentation page you provided. Could you help clarify what type of software 2FA they use?